analysis/paper/jcaa-paper.Rmd

```diff
@@ -39,19 +39,20 @@ library(Hmisc)
 
-In archaeology we have seen claims for paradigm shifts in several areas. The most wide-ranging application is Clark's [-@Clark_1993] comparison of North American archaeology to European archaeology during the 1970s-1990s. Clark, similarly to Binford and Sabloff [-@binford1982paradigms], identified the North American archaeological paradigm as rooted in anthropology, founded on post-positivist biases, and distinctive with its critical-realist ontology, a modified-objectivist epistemology, and an experimental-manipulative method. In contrast, the European archaeological paradigm is described a kind of history, mostly in a strict empiricist tradition, and typified by a realist ontology, an inductivist epistemology, and by an observational method. Clark's analysis describes a multi-paradigm situation where many research communities (i.e. North American and European) simultaneously operate in different paradigms (which Kuhn noted as a sign of a developing or immature discipline). Clark's account is not a complete Kuhnian analysis because it lacks a revolutionary change event where one paradigm is replaced by another, although he hints that processual archaeology may be considered a radical discontinuity relative to the culture-historical approaches that preceded it [@Clark_1993, pp. 206].
```

Comment:
😭 what a shame, I was a little bit proud of those very terse descriptions of N American vs European style archaeology! Never mind, maybe I can find a use for them in another paper :)